### DISCUSSION TITLE
how to get users count ?

### DISCUSSION THREAD

**Discussion by huanlirui** [logto-io/logto#2131] [Q&A]
![image](https://user-images.githubusercontent.com/40491203/195268850-c01b0b6c-0d09-488a-9d4f-3296b6d76903.png)
Cannot find the relevant api

**Comment by huanlirui** [comment]
if i dont know total count , how to set pagination ?

**Comment by huanlirui** [comment]
Hidden in the response head  ！！！ No documents and tips ！！

**Reply by gao-sun** [reply]
@wangsijie mind take a look on this?
